README.md

```diff
@@ -54,6 +54,7 @@ Hello Microverse Project is an entry level project for recent Micronuts. The aim
 - Display text on Chrome or any v8 engines
 - Mobile portfolio website
 - Error free
+- Animations and transitions
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
As per the project requirements, please add your deployed project link to your readme file🙏
<img width="625" alt="image" src="https://user-images.githubusercontent.com/37116322/229587528-ae9f5ab2-c803-47cd-b840-af6693acef5d.png">